docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // placeholder="blur" // Optional blur-up while loading
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
-
-export default Home
 ```
 
-## Image Imports
-
-You can statically `import` images that live in your project. Dynamic `await import()` or `require()` are _not_ supported.
+### Remote Images
 
-With static `import`s, you only need to provide the `src` prop. The `width`, `height`, and `blurDataURL` props will automatically be populated. Alt text is still needed separately.
+To use a remote image, the `src` property should be a URL string, which can be [relative](#loaders) or [absolute](#domains). Because Next.js does not have access to remote files during the build proccess, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and optional [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually:
 
-```js
+```jsx
 import Image from 'next/image'
-import profilePic from '../public/me.png'
 
-function Home() {
+export default function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src={profilePic}
+        src="/me.png"
         alt="Picture of the author"
-        // width={500} automatically provided
-        // height={500} automatically provided
-        // blurDataURL="data:..." automatically provided
-        // placeholder="blur" // Optional blur-up while loading
+        width={500}
+        height={500}
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
 ```
 
-For remote images, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually.
-
-## Properties
-
-[View all properties](/docs/api-reference/next/image.md) available to the `next/image` component.
-
-## Configuration
-
-In addition to [using properties](/docs/api-reference/next/image.md) available to the `next/image` component, you can optionally configure Image Optimization for more advanced use cases via `next.config.js`.
+> Learn more about the [sizing requirements](#image-sizing) in `next/image`.
 
 ### Domains
 
-To enable Image Optimization for images hosted on an external website, use an absolute url for the Image `src` and specify which
-`domains` are allowed to be optimized. This is needed to ensure that external urls can't be abused. When `loader` is set to an external image service, this option is ignored.
-
-```js
-module.exports = {
-  images: {
-    domains: ['example.com'],
-  },
-}
-```
-
-### Loader
+Sometimes you may want to access a remote image, but still use the built-in Next.js Image Optimization API. To do this, simply leave the `loader` at it's default setting and enter an absolute url for the Image `src`.
```

Comment:
Typo: it's default setting => its default setting

Reply:
Also `url` should be capitalized (I see several of these)

Reply:
Fixed